spec/config.js

```diff
@@ -1,37 +1,77 @@
 (function() {
     'use strict';
 
-    var CREDS =  {
-        'appId': 29650,
-        'authKey': 'WULOyezrmxpOgQ-',
-        'authSecret': 'TqQmBFbANJ6cfu4'
-    };
-
-    var QBUser1 = {
-            'id': 23286054,
-            'login': "js_jasmine11",
-            'password': "js_jasmine11",
-            'email': "js_jasmine11@quickblox.com"
+
+    var NEW_SERVER = false;
```

Comment:
This is our new implementation with fully clustered Chat server solution, will be released in some near future.